i3-wk-switch.py

```diff
@@ -56,10 +56,10 @@ def get_active_outputs():
     return [outp for outp in i3.get_outputs() if outp.active]
 
 
-def get_workspace(num):
-    """Returns workspace with num or None of it does not exist"""
+def get_workspace(workspace_name):
+    """Returns workspace with workspace_name or None of it does not exist"""
     want_workspace_cands = [wk for wk in i3.get_workspaces()
-                            if wk.num == num]
+                            if wk.num == workspace_name]
```

Comment:
I tested this branch, and it seem not to work for me anymore (even with numbers)
I guess it is because `wk.num` is an integer and even if `workspace_name` is a number the type in python is a string and will cause a mismatch.

Reply:
You're right--I was finally able to test with multiple monitors and saw it didn't work. It *should* work now.